client/app/bundles/course/assessment/pages/AssessmentStatistics/index.tsx

```diff
@@ -0,0 +1,169 @@
+  },
+  fetchFailure: {
+    id: 'course.assessment.statistics.fail',
+    defaultMessage: 'Failed to fetch statistics.',
+  },
+  fetchAncestorsFailure: {
+    id: 'course.assessment.statistics.ancestorFail',
+    defaultMessage: 'Failed to fetch past iterations of this assessment.',
+  },
+  fetchAncestorStatisticsFailure: {
+    id: 'course.assessment.statistics.ancestorStatisticsFail',
+    defaultMessage: "Failed to fetch ancestor's statistics.",
+  },
+  duplicationHistory: {
+    id: 'course.assessment.statistics.duplicationHistory',
+    defaultMessage: 'Duplication History',
+  },
+  marksPerQuestion: {
+    id: 'course.assessment.statistics.marksPerQuestion',
+    defaultMessage: 'Marks Per Question',
+  },
+  gradeDistribution: {
+    id: 'course.assessment.statistics.gradeDistribution',
+    defaultMessage: 'Grade Distribution',
+  },
+  submissionTimeAndGrade: {
+    id: 'course.assessment.statistics.submissionTimeAndGrade',
+    defaultMessage: 'Submission Time and Grade',
+  },
+  includePhantom: {
+    id: 'course.assessment.statistics.includePhantom',
+    defaultMessage: 'Include Phantom Student',
+  },
+});
+
+const AssessmentStatisticsPage: FC = () => {
+  const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState('marksPerQuestion');
+  const [includePhantom, setIncludePhantom] = useState(false);
+
+  const { assessmentId } = useParams();
+  const parsedAssessmentId = parseInt(assessmentId!, 10);
+  const dispatch = useAppDispatch();
+
+  const statistics = useAppSelector(getAssessmentStatistics);
+
+  useEffect(() => {
+    dispatch(actions.reset());
+    dispatch(fetchAssessmentStatistics(parsedAssessmentId)).catch(() =>
+      toast.error(t(translations.fetchFailure)),
+    );
+  }, [dispatch]);
+
+  const tabComponentMapping = {
+    marksPerQuestion: (
+      <StudentMarksPerQuestionTable includePhantom={includePhantom} />
+    ),
+    gradeDistribution: <MainGradesChart includePhantom={includePhantom} />,
+    submissionTimeAndGrade: (
+      <MainSubmissionTimeAndGradeStatistics includePhantom={includePhantom} />
+    ),
+    duplicationHistory: <DuplicationHistoryStatistics />,
+  };
+
+  if (statistics.isLoading) {
+    return <LoadingIndicator />;
+  }
+
+  return (
+    <Page
+      backTo={statistics.assessment?.url}
+      className="space-y-5"
+      title={t(translations.header, {
+        title: statistics.assessment?.title ?? '',
+      })}
+    >
+      <>
+        <Box className="max-w-full border-b border-divider">
+          <SubmissionStatusMainAssessment includePhantom={includePhantom} />
+          <FormControlLabel
+            className="mt-2"
+            control={
+              <Switch
+                checked={includePhantom}
+                className="toggle-phantom"
+                color="primary"
+                onChange={() => setIncludePhantom(!includePhantom)}
+              />
+            }
+            label={
+              <b>
```

Comment:
It might change the interface, though, if <b> is deleted (though, it will not be a major change since the sentence "include Phantom Student" won't be bold anymore).. I have removed <b> here

Reply:
you can use Typography and font-bold?

Reply:
done